source/connect/connect-from-the-command-line.txt

```diff
@@ -38,23 +38,47 @@ Command Line Connection Specification
 -------------------------------------
 
 The command line invocation for |compass-short| has two components, the 
-path to the |compass-short| executable and a connection string. The
+path to the |compass-short| executable and a connection string, you can
+optionally pass in the username and password parameters. The
 format is:
 
 .. code-block:: shell
    :copyable: false
- 
-    <path/to/compass/executable> <connection string>
 
-This example is like the basic connection string for a `MongoDB
+    <path/to/compass/executable> <connection string> 
+    --username <username> --password <password>
+
+
+.. note:: 
+
+   If the username and password arguments are not provided, Compass will
+   attempt to use the credentials in the connection string.
+
+
+Basic Connection String
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example uses a basic connection string for a `MongoDB
 University <https://learn.mongodb.com/>`__ training cluster. Modify the
 connection details to connect to your MongoDB installation:  
 
 .. code-block:: shell
-   :copyable: false
+   :copyable: true
```

Comment:
This should be false. The string won't work for the user's installation

Reply:
That might work. When I wrote the original I tried using all Xs but it didn't seem realistic enough. Maybe something like this is a happy medium:

mongodb+srv://cluster0.XXXXXX.mongodb.net/library